AnnoyancesFilter/sections/mobile-app.txt

```diff
@@ -29,7 +29,6 @@ kakuyomu.jp###readWithApp
 kakuyomu.jp##.widget-dynamicLinksButton
 1010dy.cc##.col-xs-1 > div > a[href="http://www.shuangshi.cc/"][target="_blank"] > img
 uteka.ru##.app-discount-mini-banner
-carnegie.ru##.section--2x + strong.letter-space
```

Comment:
Why removed? It is used here: https://carnegie.ru/?lang=ru

Reply:
Merged PR and restored this rule.

Reply:
And thanks)

Reply:
I removed that by mistake. sorry :(